ts/components/ui/FooterWithButtons.tsx

```diff
@@ -41,7 +41,11 @@ export default class FooterWithButtons extends React.Component<
 > {
   public render() {
     return (
-      <View style={styles.container} accessible={this.props.accessible}>
+      <View
+        style={styles.container}
+        accessible={this.props.accessible}
+        pointerEvents="box-none"
```

Comment:
```suggestion
        pointerEvents={"box-none"}
```
it's pretty the same but, as code style, we use parenthesis